src/app/components/Timestamp/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+import { number } from 'prop-types';
+import { T_BREVIER } from '../../lib/constants/typography';
+import {
+  C_RHINO,
+  GEL_SPACING_DBL,
+  FF_NEWS_SANS_REG,
+} from '../../lib/constants/styles';
+
+// if the date is invalid return null - https://stackoverflow.com/questions/1353684/detecting-an-invalid-date-date-instance-in-javascript#answer-1353711
+const isValidDateTime = dateTime => !isNaN(dateTime); // eslint-disable-line no-restricted-globals
+
+const formatDateTime = dateObj => {
+  const fullYear = dateObj.getFullYear();
+  const monthTwoDigit = dateObj.toLocaleDateString('en-GB', {
+    month: '2-digit',
+  });
+  const dayTwoDigit = dateObj.toLocaleDateString('en-GB', {
+    day: '2-digit',
+  });
+
+  return `${fullYear}-${monthTwoDigit}-${dayTwoDigit}`;
+};
+
+const formatTimestamp = dateObj =>
+  dateObj.toLocaleDateString('en-GB', {
+    day: 'numeric',
+    month: 'long',
+    year: 'numeric',
+  });
+
+const StyledTimestamp = styled.span`
+  ${T_BREVIER};
+  color: ${C_RHINO};
+  font-family: ${FF_NEWS_SANS_REG};
+  padding-bottom: ${GEL_SPACING_DBL};
+`;
+
+const Timestamp = ({ timestamp }) => {
+  const dateObj = new Date(timestamp * 1000);
+
+  if (!isValidDateTime(dateObj)) {
+    return null;
+  }
+
+  return (
+    <StyledTimestamp>
+      <time dateTime={formatDateTime(dateObj)}>{formatTimestamp(dateObj)}</time>
```

Comment:
The attribute should be all lowercase: `datetime`. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time#Attributes

Reply:
Apologies! It's an attribute that React accepts in the camel-cased form. https://reactjs.org/docs/dom-elements.html

Reply:
As spoke about offline React handles the attributes as `dateTime`. Screenshot of the rendered output for future reference:

![image](https://user-images.githubusercontent.com/7791726/47434346-4b1e1c00-d79a-11e8-8529-5e3bd10f5266.png)